Extension/src/LanguageServer/client.ts

```diff
@@ -2614,6 +2639,40 @@ export class DefaultClient implements Client {
         this.model.codeAnalysisTotal.Value = total;
     }
 
+    private async insertDoxygenComment(result: GenerateDoxygenCommentResult): Promise<void> {
+        const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+        if (editor) {
+            const oldVersion: number = result.fileVersion;
+            const newVersion: number | undefined = openFileVersions.get(editor.document.uri.toString());
+
+            if (newVersion !== undefined && oldVersion !== undefined && oldVersion === newVersion &&
```

Comment:
Can oldVersion even be undefined? It's not part of the type list for it.

Reply:
if I do not check whether the oldVersion, I will receive error message like this: 
![image](https://user-images.githubusercontent.com/106351321/181844010-deda4d57-8fff-4cb3-a31e-3ab77419b9d7.png)

Reply:
As long as you first check that newVersion is not undefined, you can compare it do oldVersion without triggering that error (without adding undefined to the type of oldVersion).  It's smart enough to understand when you've already confirmed a variable is not undefined.